src/containers/ClusterDetail/index.jsx

```diff
@@ -65,14 +114,128 @@ export default class ClusterDetail extends Component {
         title: 'Date Created', dataIndex: 'created', key: 'created', width: '13%', render: getParseDate,
       },
       {
-        title: 'Operation', dataIndex: 'operation', key: 'operation', width: '6%', render: () => '...',
+        title: 'Operation', dataIndex: 'operation', key: 'operation', width: '6%', render: (text, node) => <Popover content={renderTableOperation(node)}>...</Popover>,
```

Comment:
text param is passed, but never used

Reply:
When there is api data, all the container page will be modified. 
Currently just to show the basic components.